scss/layout.scss

```diff
@@ -23,3 +23,8 @@ body {
 .categories-and-latest.categories-and-latest .column.categories {
   margin: 0;
 }
+
+// Make easier for development
+.create-topics-notice {
```

Comment:
I'm ok with this, but maybe the actual solution is to disable these messages on Discourse? I'm not sure if this is possible.